devicon.json

```
@@ -8945,5 +8945,14 @@
         },
         "color": "#F04E98",
         "aliases": []
+    },
+    {
+      "name": "githubactions",
```

Altnames and tags are missing

You can read about what they do in our wiki page: [Updating `devicon.json`](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

You can also see a full example in this wiki page: [Example of Submitting Icons to the Repository](https://github.com/devicons/devicon/wiki/Example-of-Submitting-Icons-to-the-Repository)
```suggestion
      "name": "githubactions",
      "altnames": [
          "GitHub Actions"
      ],
      "tags": [
          "devops",
          "integration"
      ], 
```